discovery-provider/src/queries/get_celery_tasks.py

```diff
@@ -27,7 +30,10 @@ def celery_tasks_prometheus_exporter():
     )
 
     for task in tasks:
-        metric.save_time({"task_name": task["name"]}, start_time=task["time_start"])
+        try:
```

Comment:
@SidSethi this is the main fix for this PR: to wrap our metric with a try/except block.

Reply:
got it 👍  once this is released, would be good to track which metrics are failing consistently

Reply:
Indeed!